documentation/IDTA-01005/modules/ROOT/nav.adoc

```diff
@@ -10,10 +10,13 @@ Illustrations:
 Plattform Industrie 4.0; Anna Salari, Publik. Agentur für Kommunikation GmbH, designed by Publik. Agentur für Kommunikation GmbH
 ////
 
+* xref:terms-definitions-and-abbreviations.adoc[Terms and Definitions]
+* xref:preamble.adoc[Preamble]
 * xref:introduction.adoc[Introduction]
-* xref:aasx.adoc[Specification]
+* xref:basic-concepts-and-leading-picture.adoc[Basic Concepts and Leading Picture]
+* xref:aasx.adoc[AASX Specification]
 * xref:summary-and-outlook.adoc[Summary and Outlook]
-* xref:./annex/nav_annex.adoc[Annex]
+* Annex
```

Comment:
why did you remove it? It looks the same, yes, but it has advantages to have this nav-file

Reply:
The file is empty. Could you explain the advatages?